tools/i18n-toolkit/README.md

```diff
@@ -0,0 +1,77 @@
+# Validation script for translations
+
+This is a translations validator script that is able to validate rules for valid translations files.
+
+### What this toolkit validates?
+
+1.  **Structure of localisation `.json` files** - Toolkit validate structure of json files based on json schema that is defined in `src/schema/localization.ts`. If file is not valid by this schema, validation immediately failed.
+2.  **Message format** - Validation of message format if is valid based on ICU. More information about using ICU can be found here https://unicode-org.github.io/icu/userguide/format_parse/messages/. First error in ICU will cause validation failed.
+3.  **HTML syntax** - Because we use html like syntax in some messages, we need to validate if this html is valid or not. Toolkit iterate all messages and if found html inside, make validation on it. First error will cause validation failed.
+4.  **Insight to report** - Special usage of localisations. For some customer we need to rename **insight** to **report**. For this case, there is an implementation which can use piped locales `|insight` and `|report` for this. So everytime insight keyword is used in message, we need to change messages into this special definition as you can se below. This toolkit validates if there are all necessary translations and are valid.
+
+### How to install it?
```

Comment:
Should we somehow mention that this should not be done inside the SDK repository? :/ (but on the other side not make it cryptic in npm package page)